src/config/countries.php

```diff
@@ -667,6 +667,7 @@
         'name' => 'Norway',
         'diallingCode' => 47,
         'locale' => 'no',
+        'systemLocale' => 'nb_NO.UTF-8',
```

Comment:
so many different formats! does GB need one of these too?

Reply:
I don't think so as we only need it to translate dates into NO and dates work fine for GB normally